src/app/components/forms/game/build/build.vue

```diff
@@ -117,288 +117,307 @@
+							</app-form-group>
+						</div>
 
+						<!--
+							Unity Right Click Menu
+						-->
+						<app-form-group
+							v-if="formModel.type === GameBuild.TYPE_UNITY"
+							name="browser_disable_right_click"
+							:label="$gettext(`dash.games.builds.form.disable_right_click_label`)"
+						>
 							<p class="help-block">
-								<translate>
-									If your game doesn't work on HTTPS you can disable this and we'll serve it over
-									HTTP instead.
-								</translate>
+								<translate>dash.games.builds.form.disable_right_click_help</translate>
 							</p>
-
-							<div class="alert alert-notice" v-if="!formModel.https_enabled">
-								<app-jolticon icon="notice" />
-								<strong>
-									<translate>It's highly recommended to get your game working on HTTPS!</translate>
-								</strong>
-								<translate>
-									In order for your game to continue working on Game Jolt we need to pop it out into
-									a new browser window when the user clicks to play it. This results in a worse user
-									experience, but ensures that your game will still continue to be available on the
-									site.
-								</translate>
-							</div>
+							<app-form-control-toggle @changed="onBuildFieldChanged" />
 						</app-form-group>
-					</div>
 
-					<!--
-					Unity Right Click Menu
-				-->
-					<app-form-group
-						v-if="formModel.type === GameBuild.TYPE_UNITY"
-						name="browser_disable_right_click"
-						:label="$gettext(`dash.games.builds.form.disable_right_click_label`)"
-					>
-						<p class="help-block">
-							<translate>dash.games.builds.form.disable_right_click_help</translate>
-						</p>
-						<app-form-control-toggle @changed="onBuildFieldChanged" />
-					</app-form-group>
-
-					<div class="alert alert-notice" v-if="model.type === GameBuild.TYPE_UNITY">
-						<app-jolticon icon="notice" />
-						<strong>
-							<translate>Most browsers have stopped supporting the Unity Web Player.</translate>
-						</strong>
-						<translate>Please consider exporting to WebGL instead.</translate>
-					</div>
+						<div class="alert alert-notice" v-if="model.type === GameBuild.TYPE_UNITY">
+							<app-jolticon icon="notice" />
+							<strong>
+								<translate>Most browsers have stopped supporting the Unity Web Player.</translate>
+							</strong>
+							<translate>Please consider exporting to WebGL instead.</translate>
+						</div>
 
-					<p v-if="isBrowserBased">
-						<app-jolticon icon="notice" />
-						<strong>
+						<p v-if="isBrowserBased">
+							<app-jolticon icon="notice" />
+							<strong>
+								<translate>
+									Be sure to remove or block ads from showing in your browser games.
+								</translate>
+							</strong>
 							<translate>
-								Be sure to remove or block ads from showing in your browser games.
+								We serve our own ads and share the revenue with developers. Domain names to block:
```

Comment:
Is "Domain names to block" correct here?

Reply:
I think so? besides this doesnt have anything to do with this PR. Probably a diff from a merge.